pkg/types/clusterconfig/cluster_config_gcp.go

```diff
@@ -169,6 +173,16 @@ var UserGCPValidation = &cr.StructValidation{
 				Validator: validateClusterName,
 			},
 		},
+		{
+			StructField: "Preemptible",
+			BoolPtrValidation: &cr.BoolPtrValidation{
+				Default: pointer.Bool(false),
+			},
+		},
+		{
+			StructField:       "OnDemandBackup",
+			BoolPtrValidation: &cr.BoolPtrValidation{},
```

Comment:
Maybe this could be not a pointer too, we can discuss live (along with the previous one)